docs/src/IASI.md

```diff
@@ -0,0 +1,208 @@
+## IASI
+
+The Infrared Atmospheric Sounding Interferometer (IASI) is an instrument on the METOP satellites. It is a hyper-spectral measuring the radiation from the atmosphere and earth in many spectral channels. These measurements are used to derive temperature and humidity profiles at high vertical resolution. This makes IASI an key data source for numerical weather prediction (NWP).
```

Comment:
hyper-spectral measuring -> hyper-spectral sensor measuring
in many channels -> in 8461 channels
used to derive temperature and humidity profiles at high vertical resolution. -> used to derive a plethora of geophysical variables (e.g. temperature and humidity profiles).
an key -> a key
(NWP) -> (NWP) and applications in atmospheric chemistry and monitoring of essential climate variables.